src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -27,12 +26,16 @@ g.test('lodMinAndMaxClamp')
 
 g.test('maxAnisotropy')
   .desc('test different maxAnisotropy values and combinations with min/mag/mipmapFilter')
-  .params([
-    ...poptions('maxAnisotropy', [-1, undefined, 0, 1, 2, 4, 7, 16, 32, 33, 1024]),
-    { minFilter: 'nearest' as const },
-    { magFilter: 'nearest' as const },
-    { mipmapFilter: 'nearest' as const },
-  ])
+  .params(u =>
+    u //
+      .beginSubcases()
```

Comment:
Is there a difference between this and `paramsSubcasesOnly`?  Do we need `paramsSubcasesOnly`?

Reply:
Oh, there is a difference here. In `.params()`, `u` is a `CaseParamsBuilder` which can also be used as a generic combinatorial generator in places other than just the return value of `.params()`'s callback. It is used here on line 33.